solutions/prob44.js

```diff
@@ -0,0 +1,16 @@
+const StrHas = (a,b,i=0)=>{
+  if(i!==a.length){
```

Comment:
you can change this to if(i==a.length){
return false;
}
if(i !== a.length) is default you need not mention it.